Makefile.am

```diff
@@ -1,3 +1,7 @@
+# Copyright (c) 2013-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
```

Comment:
Most `Makefile`s (besides probably `src/Makefile.leveldb.include`) are modified by us already, so we should probably add a `// Copyright (c) 2014-2017 The Dash Core developers` line as well (note: this comment applies to files modified in a26bad09b6a31220583164f78efbbb1e5de2525e).